csharp_dotnetcore/60.ConversationHistory/EchoBotWithCounter.csproj

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Can this file be deleted? Seems like it can.

Reply:
Yes

Reply:
Yes :)

---
In reply to: [218234570](https://github.com/Microsoft/BotBuilder-Samples/pull/391#discussion_r218234570) [](ancestors = 218234570)